docs/quickstart.rst

```diff
@@ -47,6 +48,9 @@ The application will be the entry point for both:
 - Launching the worker that will consume the jobs created from those tasks.
 - Applying Database schema
 
+The ``App.open`` method is called to initiate the connection to the underlying
+database. It will be automatically terminated upon garbage collection.
```

Comment:
It is not completely correct to say that `App.open` initiates the connection to the underlying database. `App.open` just creates the connection pool.

Reply:
True, and I made the same confusion the the app.rst file. I correct that in both :+1:

Reply:
:+1: